tests/testthat/test-rng.R

```diff
@@ -129,6 +129,20 @@ test_that("binomial random numbers from floats have correct distribution", {
 })
 
 
+test_that("binomial random numbers from floats have correct distribution", {
+  test_rbinom_double <- function(i, m, n, p) {
+    dust_rng$new(1, 1)$rbinom(m, n, p)
+  }
+  m <- 100000
+  n <- 100L
+  p <- 0.1
```

Comment:
Do one test at np < 10, one at np > 10?

Reply:
that exists already - see block above with

```
  m <- 100000
  n <- 958L
  p <- 0.004145
```